bundles/org.openhab.binding.daikin/README.md

```diff
@@ -65,7 +65,7 @@ daikin.items:
 
 ```
 Switch DaikinACUnit_Power { channel="daikin:ac_unit:living_room_ac:power" }
-Number:Temperature DaikinACUnit_SetPoint { channel="daikin:ac_unit:living_room_ac:setpoint" }
+Number:Temperature DaikinACUnit_SetPoint { channel="daikin:ac_unit:living_room_ac:settemp" }
```

Comment:
Since this is a breaking change: is the renaming really necessary?

Reply:
this is fixing a typo in README.md - from DaikinBindingConstants.java:
public static final String CHANNEL_AC_TEMP = "settemp";